scss/homepage.md

```diff
@@ -79,4 +83,28 @@ Here is an example of what a the CSS setup in the `block.json` might look like.
     }
   }
 }
-```
\ No newline at end of file
+```
+### Custom Block Developers
+If you are creating custom blocks for your site and plan on using news-theme-css as a dependency, 
+then in addition to configuring blocks.json as per the section above, you will need to follow some additional
+steps to ensure this framework is available in your code.  There are two different senarios in custom development
+that pertain to using this framework:  Developing custom components in a theme pack and developing custom
```

Comment:
Since we are pointing out there are two things, maybe making it a list would make it a bit easier to read?